src/main/java/com/okta/sdk/framework/ApiClient.java

```diff
@@ -39,6 +40,7 @@
     protected String token;
     protected ApiClientConfiguration configuration;
     private final Logger LOGGER = LoggerFactory.getLogger(ApiClient.class);
+    private HttpResponse lastHttpResponse;
```

Comment:
I don't think so, because different instances of ApiClient could technically be talking to different orgs (and thus have different rate limit statistics). I think this is good as a regular instance variable.